src/components/userFeedback/UserFeedback.tsx

```diff
@@ -0,0 +1,63 @@
+/// <reference path="../../../types/react-tether/index.d.ts" /> Required to make dts-generator bundle react-tether definition file.
+
+import * as React from 'react';
+import { extend } from 'underscore';
+
+export interface IUserFeedbackProps {
+  feedbackText: string;
+  state: string;
+  constantClasses?: string;
+  displayOnShow?: string;
+}
+
+export interface IUserFeedbackStyle {
+  display: string;
+  className: string;
+}
+
+export const userFeedbackState = {
+  VALID: 'VALID',
+  WARNING: 'WARNING',
+  ERROR: 'ERROR',
+};
+
+export class UserFeedback extends React.Component<IUserFeedbackProps, any> {
+
+  private getDefaultUserFeedbackStyle(): IUserFeedbackStyle {
+    return {
+      display: 'none',
```

Comment:
I think I'd rather have classes passed as props depending on what we want. Then we can style both differently if we want to. There should already be styles for that in the vapor repository. If not, you can add them.